routes/sites.js

```diff
@@ -107,7 +114,7 @@ router.post('/', async (req, res) => {
     isNumeric(areaId, 'Area Id must be a Number');
     isPhoneNumber(primaryContactInfo.phone, 'Invalid Primary Phone Number');
     isBoolean(active, 'Active is not a boolean');
-    if (secondContactInfo.phone) {
+    if (secondContactInfo) {
```

Comment:
what's the purpose of this change

Reply:
There's an error if secondContactInfodoesn't exist (no attribute phone on undefined)